### PR TITLE
Adds navbar toggle button

### PR DESCRIPTION
**Pull request by bronsenivulich**

### CODE REVIEW

**Review by tim-ings — APPROVED** [review]

File: client/src/App/index.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 import { Switch, Route } from 'react-router-dom'
 import routes from 'router'
+import { Navbar } from '../Components/Navbar'
```
Comment (JeremiahPinto) [inline client/src/App/index.tsx:4]: I've set up a config so you can just use `Components/Navbar`

File: client/src/App/index.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 const App = () => {
   return (
     <div className='App'>
+      <Navbar />
       <header>foodbank</header>
```
Comment (JeremiahPinto) [inline client/src/App/index.tsx:10]: delete this line

File: client/src/Components/Navbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './navbar.css';
```
Comment (JeremiahPinto) [inline client/src/Components/Navbar/index.tsx:2]: Please use the [tailwindcss](https://tailwindcss.com/docs/display) classes.

File: client/src/Components/Navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+export const NavbarContent = () => {
+  return <div>navbar here</div>
+};
+
+export const Navbar: React.FC = () => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div>
```
Comment (JeremiahPinto) [inline client/src/Components/Navbar/index.tsx:12]: Use `header` here. Try to stick with semantic HTML where possible

File: client/src/Components/Navbar/navbar.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  .navbar-toggle-button {
+    display: block;
+  }
+  .navbar-container {
+    display: none;
+  }
+  .navbar-container.open {
+    display: block;
+  }
+}
```
Comment (JeremiahPinto) [inline client/src/Components/Navbar/navbar.css:19]: This file can be deleted when tailwind is used

**Review by JeremiahPinto — CHANGES_REQUESTED** [review]
I understand this may be early, but it's good so you can get feedback early

**Comment by JeremiahPinto** [comment]
I am definitely confused, is this Tim or Bronsen working on it.

**Comment by Drewbi** [comment]
> I am definitely confused, is this Tim or Bronsen working on it.

Team effort 😆

**Comment by tim-ings** [comment]
We paired on Saturday, figured I'd fix the remaining issues here so this can be merged and Bronsen is free to get started on the next part of his ticket.

**Review by JeremiahPinto — APPROVED** [review]

**Pull request merged**